pkg/vault/test/authorizer/TimelockAuthorizerExecute.test.ts

```diff
@@ -78,6 +79,32 @@ describe('TimelockAuthorizer execute', () => {
       return authorizer.schedule(where, data, executors || [], { from: user });
     };
 
+    it('increases the scheduled execution count', async () => {
+      const countBefore = await authorizer.instance.getScheduledExecutionsCount();
+      await schedule(authenticatedContract);
+      const countAfter = await authorizer.instance.getScheduledExecutionsCount();
+
+      expect(countAfter).to.equal(countBefore.add(1));
+    });
+
+    it('stores scheduler information', async () => {
+      const id = await schedule(authenticatedContract);
+
+      const scheduledExecution = await authorizer.getScheduledExecution(id);
+      expect(scheduledExecution.scheduledBy).to.equal(user.address);
```

Comment:
minor detail: the fact that `user` is scheduling the execution is not obvious just from reading this diff. This is not the case in the other tests added in this PR.

Do you think it would make sense to make it a bit more explicit?

Reply:
I'm reusing the `schedule` function, which is defined 10 lines above, and which clearly sends the tx from `user`. All of the other tests in this file make that clear. I think this is fine.